data/ka'het/ka'het outfits.txt

```diff
@@ -197,12 +291,24 @@ outfit "Ka'het Shield Restorer"
 	category "Systems"
 	cost 138000
 	thumbnail "outfit/ka'het shield restorer"
-	"mass" 15
-	"outfit space" -19
+	"mass" 14
+	"outfit space" -17
 	"shield generation" 1.02
-	"shield energy" 2.04
+	"energy consumption" 1.53
 	description "The Builders never relied much on shield matrices, preferring instead to strengthen their hull. The reality of war, however, required that the Builders to develop strong shield technology, leading to some Het vessels making extensive use of this module."
 
+outfit "Ka'het Grand Restorer"
+	category "Systems"
+	cost 1022000
+	thumbnail "outfit/ka'het grand restorer"
+	"mass" 72
+	"outfit space" -86
+	"shield generation" 7.08
+	"energy consumption" 10.62
+	"depleted shield delay" 350
+	description "This shield generator is the apex of Ka'het shielding technology, a field that developed rapidly and out of necessity rather than perfected over long periods. Nevertheless, the Grand Restorer grants good protection for the largest Het ships, replacing the basic restorers used elsewhere."
```

Comment:
```suggestion
	description "This shield generator is the apex of Ka'het shielding technology, a field that developed rapidly and out of necessity rather than gradually and perfected over long periods. Nevertheless, the Grand Restorer grants good protection for the largest Het ships, replacing the basic restorers used elsewhere."
```
While you had `perfected over long periods` mirror `out of necessity`, you didn't have a mirror for `rapidly`. I added `gradually` to give `rapidly` a parallel. Does that make sense? Not sure if that was the best way to explain it.
Besides that, there might be a better word choice for `good`, but I can't think of something else. If you've another word in mind that you could substitute, consider switching it out.